packages/rocketchat-lib/server/functions/saveUser.js

```diff
@@ -130,6 +135,10 @@ RocketChat.saveUser = function(userId, userData) {
 			updateUser.$set.name = userData.name;
 		}
 
+		if (userData.role) {
+			updateUser.$set.roles = [ (userData.role || 'user') ];
+		}
+
```

Comment:
@rodrigok 

I thought to override the role since there is no UI to assign multiple role besides a button to make user admin.

@RocketChat/core thoughts?